source/macos/ADCMacOSVisualizer/ADCMacOSVisualizer/RootViewController.xib

```diff
@@ -126,28 +126,18 @@
                         <autoresizingMask key="autoresizingMask"/>
                     </scroller>
                     <scroller key="verticalScroller" wantsLayer="YES" verticalHuggingPriority="750" horizontal="NO" id="CYT-4i-Rnq">
-                        <rect key="frame" x="334" y="0.0" width="16" height="351"/>
+                        <rect key="frame" x="334" y="0.0" width="16" height="376"/>
                         <autoresizingMask key="autoresizingMask"/>
                     </scroller>
                 </scrollView>
                 <comboBox verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="O4M-uk-PGq">
-                    <rect key="frame" x="873" y="7" width="154" height="23"/>
+                    <rect key="frame" x="750" y="6" width="259" height="23"/>
                     <comboBoxCell key="cell" scrollable="YES" lineBreakMode="clipping" selectable="YES" sendsActionOnEndEditing="YES" borderStyle="bezel" placeholderString="Select Host Config" bezelStyle="round" completes="NO" usesDataSource="YES" numberOfVisibleItems="5" id="xyg-WJ-FcE">
                         <font key="font" metaFont="system"/>
                         <color key="textColor" name="controlTextColor" catalog="System" colorSpace="catalog"/>
                         <color key="backgroundColor" name="textBackgroundColor" catalog="System" colorSpace="catalog"/>
                     </comboBoxCell>
                 </comboBox>
-                <button verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="DZe-UF-Cz8">
-                    <rect key="frame" x="909" y="26" width="78" height="32"/>
-                    <buttonCell key="cell" type="push" title="Render" bezelStyle="rounded" alignment="center" borderStyle="border" imageScaling="proportionallyDown" inset="2" id="4FA-yZ-Lh4">
-                        <behavior key="behavior" pushIn="YES" lightByBackground="YES" lightByGray="YES"/>
-                        <font key="font" metaFont="system"/>
-                    </buttonCell>
-                    <connections>
-                        <action selector="handleRenderAction:" target="-2" id="OJB-HB-Iqa"/>
-                    </connections>
-                </button>
```

Comment:
Why did you remove this?

Reply:
I have changed the position for buttons.I forgot to add new image, let me add new image.